test_vm/tests/replica_update_test.rs

```diff
@@ -719,13 +720,12 @@ fn extend_after_upgrade() {
         st.sectors = sectors.amt.flush().unwrap();
     });
 
-    let extension_epoch = v.get_epoch();
     let extension_params = ExtendSectorExpirationParams {
         extensions: vec![ExpirationExtension {
             deadline: deadline_index,
             partition: partition_index,
             sectors: make_bitfield(&[sector_number]),
-            new_expiration: extension_epoch + policy.max_sector_expiration_extension - 1,
+            new_expiration: activation + policy.max_sector_expiration_extension - 1,
```

Comment:
How was this failing?  It seems like it should continue working unchanged that we can extend with max expiration extension from the extension epoch and not the activation epoch which is <= extension epoch

Reply:
The max extension is now the max sector lifetime.